### ISSUE TITLE
Edit Question: ua_ruhr_v1.1.0-beta.1/technical-classification/data-tools/creation_methods

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#59]
**Describe the request**  
Add software version in the help text, as this is important information

**Proposed setting or content**   
*help text ger*  
"... allerdings müssen alle Geräte, Software, **Softwareversion** und auch Informationen über die Vorgehensweise erhalten bleiben..."

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] help text (ger)

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/80

**Comment by ClaraRosalie** [comment]
implemented in upstream rdmorganiser#82
